### PR TITLE
feat: Add gcp_sql_instances to the resources view

### LINKED ISSUE #373: CQ Resource View
`gcp_sql_instances` not included

### PR DESCRIPTION
**Pull request by irmatov**
Closes #373

### CODE REVIEW

File: views/resource.sql [hunk 1]
```diff
@@ -1,20 +1,20 @@
 DROP VIEW IF EXISTS gcp_resources;
```
Comment (irmatov) [inline views/resource.sql:1]: Only case changes in this file (transformed to uppercase).

File: resources/services/sql/sql_instances.go [hunk 2]
```diff
@@ -115,9 +115,10 @@ func SQLInstances() *schema.Table {
 				Type:        schema.TypeBigInt,
 			},
 			{
-				Name:        "name",
-				Description: "Name of the Cloud SQL instance This does not include the project ID",
+				Name:        "id",
```
Comment (irmatov) [inline resources/services/sql/sql_instances.go:118]: `name` is actually an identifier for SQL instances according to the docs: https://cloud.google.com/sql/docs/mysql/instance-settings

Renaming `name` to `id` allows it to be included in the `gcp_resources` view (`id` field is required).

Comment (roneli) [reply resources/services/sql/sql_instances.go:118]: I would keep both, as someone one reading  wouldn't expect this transformation

**Review by roneli — COMMENTED** [review]
Good catch, tricky field, I would keep both as some users will expect the original field, ill let more ppl chime in opinion, but thats my thoughts

**Review by bbernays — APPROVED** [review]

**Pull request merged**